app/http.php

```
@@ -93,7 +94,7 @@
         try {
             Console::success('[Setup] - Creating database: appwrite...');
             $dbForConsole->create();
-        } catch (\Throwable $e) {
```

why make it specific to Duplicate? What if there are some other kinds of exceptions? Its gonna be uncaught right?

---

Yep but if it's something else, we shouldn't be swallowing the exception and just printing that the table exists